themes/classic/_dev/css/partials/_variables.scss

```diff
@@ -1,2 +1,6 @@
-$gray: #3D3C44;
-$gray-light: #b1b1b1;
+$gray-dark: #3D3C44;
+$gray: #b1b1b1;
+$gray-light:#d2d2d2;
+$gray-lighter:#f2F2F2;
```

Comment:
gray lighter than lighter etc. why not just use few motive colors? default-bootstrap was created in a same way, more variables is not better!